ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/DevelopmentCcdConfig.java

```diff
@@ -9,13 +11,18 @@
 
 public class DevelopmentCcdConfig implements CCDConfig<CaseData, State, UserRole> {
 
+    @SneakyThrows
     @Override
     public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuilder) {
-
         configBuilder.setEnvironment("development");
         configBuilder.setWebhookConvention(this::webhookConvention);
 
-        new CcdConfigApplier(new NoFaultDivorceCcdConfigFactory()).applyTo(configBuilder);
+        Reflections reflections = new Reflections("uk.gov.hmcts.reform.divorce.ccd");
```

Comment:
The reason I setup a Factory for the configuration was to make it easy to add other Case types in the future.  We may need to revert to that method or be more creative with the package names if we use this reflection method.

Reply:
that's fair enough, I think it might be best to have a whole other gradle project for that